packages/block-library/src/legacy-widget/index.php

```diff
@@ -69,6 +69,9 @@ function register_block_core_legacy_widget() {
 				'isCallbackWidget' => array(
 					'type' => 'boolean',
 				),
+				'hasEditForm'      => array(
```

Comment:
It seems odd to me to call this an attribute of the block, considering that it's of no consequence to the saved result or semantic meaning of a block's content. It seems like it's being used this way as a convenience?